src/Model/Table/QueuedTasksTable.php

```diff
@@ -67,11 +67,11 @@ public function createJob($jobName, $data = null, $notBefore = null, $group = nu
 		$data = [
 			'jobtype' => $jobName,
 			'data' => serialize($data),
-			'group' => $group,
+			'task_group' => $group,
 			'reference' => $reference
 		];
 		if ($notBefore !== null) {
-			$data['notbefore'] = date('Y-m-d H:i:s', strtotime($notBefore));
+			$data['notbefore'] =  strtotime($notBefore);
```

Comment:
why is this necessary? does the ORM better work with int instead of date strings? maybe we should use Time() object rather?